docs/extensions/rascaline_json_schema.py

```diff
@@ -1,10 +1,9 @@
-import os
 import json
+import os
 
-from docutils.parsers.rst import Directive
-from docutils import nodes
 import myst_parser.main
-
+from docutils import nodes
```

Comment:
I would put docutils as a known third-party

Reply:
Done.

Reply:
Should be done with the next commit.